### ISSUE TITLE
pyproject.toml autopandas=true turns CTEs to None 

### ISSUE THREAD

**Issue by lfunderburk** [ploomber/jupysql#790]
Since updating to jupysql 0.9.0 I have been getting mixed results when transforming results to pandas format

What I expected:

Setting `autopandas=true` would allow me to set all my query results to pandas dataframe

What happened:

When `autopandas=true`, the query returned `None`

When `autopandas=false`, the query returned expected results

When `autopolars = true`, the query returned expected results

## Data download 

Location to banking script 
https://github.com/ploomber/sql/blob/main/banking.py 

```
import sys

sys.path.insert(0, "../../../")
import banking  # noqa: E402

_ = banking.BankingData("https://tinyurl.com/jb-bank", "bank")
_.extract_to_csv()

_ = banking.MarketData("https://tinyurl.com/jb-bank-m", "expanded_data")
_.extract_asc_to_csv()
```

## Pyproject toml

```
[tool.jupysql.SqlMagic]
displaycon = false
feedback = true
autopandas = false
autopolars  = false
```

## Init duck db 
```
%reload_ext sql
%sql duckdb:///bank.duck.db
```

## Create schema

```
%%sql
CREATE SCHEMA s1;
CREATE TABLE s1.account AS
FROM read_csv_auto('expanded_data/account.csv', header=True, sep=',');
CREATE TABLE s1.district AS
FROM read_csv_auto('expanded_data/district.csv', header=True, sep=',');
CREATE TABLE s1.loan AS
FROM read_csv_auto('expanded_data/loan.csv', header=True, sep=',');
```

When `autopandas=false`

and 

```
%%sql --save loan_amount_district_id
SELECT l.loan_id, 
        l.amount, 
        a.district_id 
FROM s1.loan AS l 
INNER JOIN s1.account AS a 
    ON l.account_id = a.account_id
```

and

```
result = %sql SELECT * FROM loan_amount_district_id
type(result)
```

Yields `sql.run.resultset.ResultSet`

If I change `autopandas = true`

```
[tool.jupysql.SqlMagic]
displaycon = false
feedback = true
autopandas = true
autopolars  = false
```

Then, reloading the extension, performing the CTE and save it to `pandas.DataFrame` via pyproject.toml, it is `None`

If I change `autopandas = false`,` autopolars  = true`

```
[tool.jupysql.SqlMagic]
displaycon = false
feedback = true
autopandas = false
autopolars  = true
```

It returns an object of type polars.dataframe but the content is empty

**Comment by lfunderburk** [comment]
Something similar happened with JupySQL 0.9.0 if I tried to transform the result to DataFrame()

**Comment by lfunderburk** [comment]
If I try autopolars=true
the type is correct, but the table comes back empty

**Comment by edublancas** [comment]
can you attach a notebook that I can run and doesn't depend on any external files? (e.g., it downloads whatever data it needs, creates the pyproject.toml, etc.).

**Comment by lfunderburk** [comment]
Hi Eduardo, 

Here is a notebook on colab

https://colab.research.google.com/drive/1anexYkxi2Mcq-U2gP169HdUinEX8ZiZA?usp=sharing

**The only thing this notebook needs is a modification to create the header of the toml file appropriately**

The error persists whether I use the toml file, or whether I use `%config SqlMagic.autopandas=True`

**Comment by edublancas** [comment]
I reproduced the problem, looks like this has been fixed already, can you try using the latest version from github?

```
!pip uninstall jupysql --yes --quiet
!pip install git+https://github.com/ploomber/jupysql --quiet
import sql; print(sql.__version__) # should print 0.9.1.dev
```

if this works, I'll make a release

**Comment by edublancas** [comment]
hey @lfunderburk: I did some research and confirmed this is a JupySQL bug introduced in 0.9.0, I'm working on a fix

**Comment by edublancas** [comment]
I just released 0.9.1 so this should be fixed. @lfunderburk if you still have issues, feel free to re-open this
